src/components/WalletSend.vue

```diff
@@ -203,18 +208,7 @@ export default class WalletSend extends Mixins(
   NetworkFeeWarningMixin
 ) {
   readonly delimiters = FPNumber.DELIMITERS_CONFIG;
-  readonly vestingPeriodsMap = {
-    1: '1 day',
-    7: '7 days',
-    30: '30 days',
-    60: '60 days',
-    90: '90 days',
-  };
-
-  readonly vestingPeriods = Object.entries(this.vestingPeriodsMap).map(([value, label]) => ({
-    value: +value as UnlockPeriodDays,
-    label,
-  }));
+  readonly vestingPeriodsInDays: UnlockPeriodDays[] = [1, 7, 30, 60, 90];
```

Comment:
Move to consts?

Reply:
already const, don't want to make it sharable since it'll be used only here